pxblue/angular-design-patterns
# Bug/138 fix styling drawer layout

Fixes [138](https://github.com/pxblue/angular-design-patterns/issues/138) .


#### Changes proposed in this Pull Request:
Removed ::ng-deep .pxb-drawer-layout-nav-content {} css from following component
- Fix Length component
- Verify on submit component
- Sectioned form component
- In a list component
- Phone number format


#### To Test:
- http://localhost:4200/forms-and-validation/fixed-length-passcode
- http://localhost:4200/forms-and-validation/in-a-list
- http://localhost:4200/forms-and-validation/in-a-table
- http://localhost:4200/forms-and-validation/phone-number-format
- http://localhost:4200/forms-and-validation/sectioned-form
- http://localhost:4200/forms-and-validation/verify-on-submit
